packages/owl-bot/src/owl-bot.ts

```diff
@@ -101,16 +109,20 @@ export = (privateKey: string | undefined, app: Probot, db?: Db) => {
       },
       context.octokit
     );
-
-    await context.octokit.issues.removeLabel({
-      name: OWLBOT_RUN_LABEL,
-      issue_number: prNumber,
-      owner,
-      repo,
-    });
+    try {
+      await context.octokit.issues.removeLabel({
+        name: OWLBOT_RUN_LABEL,
+        issue_number: prNumber,
+        owner,
+        repo,
+      });
+    } catch (err) {
+      logger.error(
```

Comment:
I'm more comfortable if we only ignore 404 (expected) and re-throw anything else?

Reply:
I just wanted to make sure that the 404 code is actually populated, then I'll tighten. Not sure what the shape of the error object is. I might test locally first and send a different PR.

Reply:
https://github.com/googleapis/repo-automation-bots/blob/83ae51929068927043a037b46b22ce7e5364dda8/packages/snippet-bot/src/snippet-bot.ts#L698

You can add a test emulating 404 response with nock.